api/src/main/java/edu/cornell/mannlib/vitro/webapp/web/templatemodels/individual/ObjectPropertyTemplateModel.java

```diff
@@ -386,6 +386,20 @@ protected String getObjectKey() {
         return objectKey;
     }
 
+    protected boolean isAuthorizedToDisplay(VitroRequest vreq, String objectUri) {
+        boolean isAuthorizedToDisplay = false; 
+        AccessObject ao;
+        if (isFaux()) {
+            ao = new FauxObjectPropertyStatementAccessObject(vreq.getJenaOntModel(), subjectUri, fauxProperty, objectUri);
+        } else {
+            ao = new ObjectPropertyStatementAccessObject(vreq.getJenaOntModel(), subjectUri, property, objectUri);
+        }
+        if (PolicyHelper.isAuthorizedForActions(vreq, ao, AccessOperation.DISPLAY) ) {
```

Comment:
Can this be simplified to just return `PolicyHelper.isAuthorizedForActions(vreq, ao, AccessOperation.DISPLAY)`?

Reply:
Fixed